datacats/scripts/test_remote_server.sh

```diff
@@ -0,0 +1,10 @@
+# run bash within datacats/web with bind mounted ssh keys&config files
+# so that you can ssh to the remote server within the docker
+
+docker run \
+    -it \
+    -v  ~/.datacats/user-profile/id_rsa:/root/.ssh/id_rsa \
+    -v  ~/datacats/datacats/scripts/known_hosts:/root/.ssh/known_hosts \
+    -v  ~/datacats/datacats/scripts/ssh_config:/etc/ssh/ssh_config \
+        datacats/web \
+            bash
```

Comment:
The files in the scripts dir are just things that we're bind mounting into containers so this doesn't fit here. Also, this seems more like a development helper that most people won't need, especially now that you've improved all the errors.